docker/Dockerfile.ipfs

```diff
@@ -0,0 +1,3 @@
+FROM ipfs/go-ipfs
```

Comment:
You can override the start command in the docker-compose.yml file via the `command` block, meaning we wouldn't need our own image for it!

https://docs.docker.com/compose/compose-file/#command